code/src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Header = () => {
+  return (
+    <h1 className="Header">
```

Comment:
The ```className="Header"``` was never used, and maybe it shouldn't be uppercase?

Reply:
You are very right! Will check this out, thanks.